sklearn/decomposition/nmf.py

```diff
@@ -402,12 +402,13 @@ def _init(self, X):
             W, H = _initialize_nmf(X, self.n_components_, variant='ar',
                                    random_state=rng)
         elif init == "random":
-            W = rng.randn(n_samples, self.n_components_)
+            scale = np.sqrt(X.std()) / self.n_components_ ** (1. / 4)
```

Comment:
IIRC, @TomDLT is planning to open a new PR for exploring different initializations once #4852 is merged.